packages/xo-server-sdn-controller/src/index.js

```diff
@@ -706,11 +706,7 @@ class SDNController extends EventEmitter {
       return
     }
 
-    const encapsulation =
-      network.other_config.encapsulation !== undefined
-        ? network.other_config.encapsulation
-        : 'gre'
-
+    const encapsulation = network.other_config.encapsulation || 'gre'
```

Comment:
This syntax is shorter but actually less safe. So when you use it, make sure you know exactly what you're doing: if `network.other_config.encapsulation` can be an empty string (or any other falsy value) as a valid value, don't use it! Because it will unexpectedly fallback to `'gre'`.
Do you confirm this is what you want to do?

Reply:
The value is either `gre` or `vxlan` (or `undefined` if not set) it can't be an empty string.
It's only set by my controller and with only these 2 values.

So yes, I think it's okay. :)

Reply:
You could have used destructuring for this case:

```js
const { encapsulation = 'gre' } = network.other_config
```

Also, keep in mind that you can use the [nullish coalescing operator](https://babeljs.io/docs/en/babel-plugin-proposal-nullish-coalescing-operator) `??` which is safer than `||`.

Reply:
Noted, I didn't know that.
Thanks for the tip. :)